airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -61,6 +62,28 @@ class TITargetStatePayload(BaseModel):
     state: IntermediateTIState
 
 
+class TIDeferredStatePayload(BaseModel):
+    """Schema for updating TaskInstance to a deferred state."""
+
+    state: Annotated[
+        Literal[IntermediateTIState.DEFERRED],
+        # Specify a default in the schema, but not in code, so Pydantic marks it as required.
+        WithJsonSchema(
+            {
+                "type": "string",
+                "enum": [IntermediateTIState.DEFERRED],
+                "default": IntermediateTIState.DEFERRED,
+            }
+        ),
+    ]
+
+    classpath: str
+    kwargs: dict[str, Any]
+    created_date: UtcDateTime
```

Comment:
Don't think we need it here, we can handle it on server side itself in `update_state`

Reply:
This is how it looks in the Trigger class:
```
    def __init__(
        self,
        classpath: str,
        kwargs: dict[str, Any],
        created_date: datetime.datetime | None = None,
    ) -> None:
        super().__init__()
        self.classpath = classpath
        self.encrypted_kwargs = self._encrypt_kwargs(kwargs)
        self.created_date = created_date or timezone.utcnow()
```

If we do not allow to override it, wont it be a regression from the past? We can just leave out the field.

Reply:
>If we do not allow to override it,

I can't think of scenarios where we would want to override the `created_at` date from the worker. So better to remove it from here and handle it in API logic i.e. in `update_state`

Reply:
Alright, removing it.
Handled in: 03e83e2fa0d3c7054eee75bff4759a5513e4dfc8